### PR TITLE
feat: prompt for feedback/discord on first new project

### LINKED ISSUE #194: Prompt users to provide feedback
Given we want valuable feedback from users of the Nitric CLI, perhaps we could show a text output after the first use of `nitric new` (or another appropriate command?) which prompts with a message like this:

```
Thanks for trying nitric!

If you have feedback you can raise issues on GitHub https://github.com/nitrictech/nitric or come talk with us directly on Discord https://our.discord.link.example
```

### PR DESCRIPTION
**Pull request by HomelessDinosaur**

### CODE REVIEW

File: pkg/cmd/root.go [hunk 1]
```diff
@@ -58,9 +64,82 @@ var rootCmd = &cobra.Command{
...
+		num, err = strconv.Atoi(string(contents))
+		if err != nil {
+			return false
+		}
+	}
+	if num == -1 { // Set as do not prompt
+		return false
+	} else if num > requiredTimes { // We can prompt
+		return true
+	} else {
```
Comment (jyecusch) [inline pkg/cmd/root.go]: Seems like this prompt will appear endlessly after the 6th usage. It seems like something that should happen once or occasionally, not constantly.

File: pkg/cmd/root.go [hunk 2]
```diff
@@ -58,9 +64,82 @@ var rootCmd = &cobra.Command{
...
+		return false
+	}
+	num := 0
+	if contents != nil {
+		num, err = strconv.Atoi(string(contents))
+		if err != nil {
+			return false
+		}
+	}
+	if num == -1 { // Set as do not prompt
```
Comment (jyecusch) [inline pkg/cmd/root.go]: This `-1` value is implicitly coupled to the `setFeedbackNum(-1)` call in the prompt function, which means if one changes they both need to change - but that's not obvious without reading all of the code.

The relationship would probably be better if it was explicit - e.g. use a constant or maybe store a JSON/YAML value in the document based on a Struct that contains the counts and user preferences.

File: pkg/cmd/root.go (hunk 1 above)
Comment (jyecusch) [reply pkg/cmd/root.go]: e.g. once per day/week

**Review by jyecusch — DISMISSED** [review]

File: pkg/cmd/root.go [hunk 3]
```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/nitrictech/cli/pkg/output"
 )
 
+const feedbackMsg = "Thanks for trying nitric!\nIf you have feedback you can raise issues on GitHub https://github.com/nitrictech/nitric or come talk with us directly on Discord https://discord.gg/EBPCEgG9"
```
Comment (tjholm) [inline pkg/cmd/root.go]: Not sure how we can sort this out, the current invite link contains your username for discord. This isn't a huge issue I think, but good to see if we could get around this.

Comment (davemooreuws) [reply pkg/cmd/root.go:33]: @tjholm @HomelessDinosaur how's this link? https://discord.gg/bvTW7aYg6B

Comment (davemooreuws) [reply pkg/cmd/root.go]: @tjholm maybe we just make a nitric user in discord for this purpose.

Comment (tjholm) [reply pkg/cmd/root.go]: Created a new invite link with my current user `tjh`: https://discord.com/invite/Webemece5C

Comment (tjholm) [reply pkg/cmd/root.go]: Will look at server boosting later to get a custom invite link going.

Comment (HomelessDinosaur) [reply pkg/cmd/root.go]: Change to https://discord.com/invite/Webemece5C, as posted on twitter

**Review by asalkeld — APPROVED** [review]

**Review by jyecusch — APPROVED** [review]

**Pull request merged**